src/main/java/xRoss.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class xRoss {
+
+    // prints a divider line and new line to command line output
+    public static void printDividerLine() {
```

Comment:
i like how you have a function just for the divider lines, makes the code at main less cluttered